reana_cluster/backends/kubernetes/templates/kerberos/config.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Let's rename the directory which contains the configuration from `kerberos` to `configmaps`.